src/main/java/jp/skypencil/errorprone/slf4j/Slf4jLoggerShouldBeNonStatic.java

```diff
@@ -38,14 +38,7 @@ public Description matchVariable(VariableTree tree, VisitorState state) {
       SuggestedFixes.removeModifiers(tree, state, Modifier.STATIC).ifPresent(builder::merge);
       suggestRename(tree, state).ifPresent(builder::merge);
 
-      return Description.builder(
-              tree,
-              "Slf4jLoggerShouldBeNonStatic",
-              "https://github.com/KengoTODA/findbugs-slf4j#slf4j_logger_should_be_non_static",
-              SUGGESTION,
-              "Do not use static Logger field, use non-static one instead")
```

Comment:
message will be taken from BugPattern summary if not explicitly set on the description